cms/templates/settings_graders.html

```diff
@@ -25,7 +27,11 @@
 </%block>
 <%block name="requirejs">
     require(["js/factories/settings_graders"], function(SettingsGradersFactory) {
-        SettingsGradersFactory(_.extend(${escape_json_dumps(course_details, cls=CourseSettingsEncoder) | n}, {is_credit_course: ${escape_json_dumps(is_credit_course) | n}}), "${grading_url}");
+        SettingsGradersFactory(
+            _.extend(${escape_jsoon_for_js(course_details, cls=CourseSettingsEncoder) | n},
```

Comment:
Unfortunately filters (the right side of the pipe) can only take a single argument (the left side of the pipe).  Since this call requires the cls argument, this needs to be an acceptable exception.